src/components/SolutionFilter/SolutionFilter.tsx

```diff
@@ -0,0 +1,100 @@
+import React, {useState} from "react";
+import ImageCard from "../ImageCard/ImageCard";
+import { Chip } from "@mui/material";
+import cslx from "clsx";
+import styles from "./SolutionFilter.module.css";
+//TODO - use MUI Stack to create container for chips
+
+//TODO - Create solution interface to include in solutions array for type checking?
+const SolutionFilter: React.FC<{ solutions: any[] }> = ({ solutions }) => {
+
+    const [selectedTags, setSelectedTags] = useState<string[]>([]);
+    const uniqueTags: string[] = [];
+
+    // Finds all unique tags in the solutions array.
+    const findTags = () => {
+        solutions.forEach((solution) => {
+            solution.tags.forEach((tag: string) => {
+                if (!uniqueTags.includes(tag)) {
+                    uniqueTags.push(tag);
+                }
+            });
+        });
+    };
+
+    // Updates the selected tags based on the tag that was clicked.
+    const updateTags = (tag: string) => {
+        if (selectedTags.includes(tag)) {
+            setSelectedTags(selectedTags.filter((t) => t !== tag));
+        } else {
+            setSelectedTags([...selectedTags, tag]);
+        }
+    };
+
+    // Renders all of the menu options based on the passed in solutions.
+    const renderFilterChips = () => {
+        findTags();
+        return uniqueTags.map((tag) => (
+            <Chip
+                label={tag[0].toUpperCase() + tag.slice(1)}
+                onClick={() => {
+                    updateTags(tag);
+                }}
+                className={cslx(styles.solutionFilterChip)}
+                sx={{
+                    fontSize: "14px",
+                    width: "100%",
+                    height: "37px",
+                    fontWeight:"500",
+                    borderRadius: "10px",
+                    color: selectedTags.includes(tag) ? "#FFF4F3" : "#AF1107",
+                    backgroundColor: selectedTags.includes(tag) ? "#AF1107" : "#FFF4F3",
```

Comment:
check the spelling of clsx you have cslx , it works since you have it uniformly spelled that way but it leaves an error.  3 places, check your import cslx from clsx.

add this line under backgroundColor.
This is the  color from the figma file. or use hex #c36f69
If you elect to use a css file then simply add !mportant to the :hover 

"&:hover": {backgroundColor: "rgba(195, 111, 105, 1) "},